Test 'test_exechost_periodic_accept' of TestHookExechostPeriodic is failing intermittently while verifying server-logs

Mahalty opened the pull request:
<!--- Please review your changes in preview mode -->
<!--- Provide a general summary of your changes in the Title above -->

#### Describe Bug or Feature

- Test 'test_exechost_periodic_accept' of TestHookExechostPeriodic is failing intermittently while verifying server-logs


#### Describe Your Change

-  In the hook script we are updating resource of a invalid vnode. Periodic hook upon encountering an invalid node ’, proceeds to create one and the test is verifying this by verifying the following msg "autocreated vnode <vnode name>". In the current test, the test is verifying the existence of this log msg and along with that it is also verifying that the msg is received from first mom. The test is failing intermittently as the log msg in failed case has been sent from the other mom. So updating the test to check just for the log msg instead of checking from which mom the msg has been recieved.
-  In the test, hook is creating the second mom has vnode due to whichin the next test run whenever the mom is created it creates the vnode(Other mom) and when it tries to create other mom it throws error "Node already exist"


#### Link to Design Doc
<!--- If there is a design, link to it here: **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)** -->


#### Attach Test and Valgrind Logs/Output
[Exec_periodic.txt](https://github.com/openpbs/openpbs/files/6782312/Exec_periodic.txt)



<!--- Pull Request Guidelines: [Pull Request Guidelines](https://pbspro.atlassian.net/wiki/spaces/DG/pages/1187348483/Pull+Request+Guidelines) -->

bayucan commented:
Code update looks fine. Please attach test log.

Mahalty commented:
@bayucan @bhroam  I have made few more changes and attached execution logs. Kindly review

bayucan commented on test/tests/functional/pbs_hook_exechost_periodic.py:
```diff
@@ -122,21 +118,18 @@ def test_exechost_periodic_accept(self):
+        exp_msg2 = msg1 + common_msg2
         msg2 = "%s;Updated vnode %s's resource " % (self.momA.hostname,
                                                     other_node)
         exp_msg3 = msg2 + common_msg2
```
I'm seeing msg1 and msg2 to be the same, so exp_msg2 and exp_msg3 are also the same.

bhroam commented on test/tests/functional/pbs_hook_exechost_periodic.py:
```diff
@@ -43,19 +43,13 @@
+node = ''
+for k in vn.keys():
+    if host in k:
+        node = k
```
If you are only searching for 1, then I'd suggest breaking.

Mahalty commented:
@bayucan  and @bhroam  I have addressed your review comments. Kindly review

Latest execution-logs:
[Exec_periodic_updated.txt](https://github.com/openpbs/openpbs/files/6788633/Exec_periodic_updated.txt)

bayucan reviewed (approved):
Looks good.

bayucan commented:
@bhroam : are you still okay with the latest commit? If so, I can merge this now.
